internal/resources/cluster/nodepools/resource_node_pool.go

```diff
@@ -565,6 +565,7 @@ func flattenTKGSVolumes(volume *nodepoolmodel.VmwareTanzuManageV1alpha1CommonClu
 		return nil
 	}
 
+	// helper.SetPrimitiveValue(volume.Capacity, flattenVolumes[capacityKey], "YYY")
```

Comment:
`flattenVolumes` is a map of string and interface, so we can't use the helper function, also the `SetPrimitiveValue` function won't work with the maps. map indices are not addressable, meaning we can’t use pointers to a map index.